### ISSUE TITLE
Vue components are not included on first build (OSOE-892)

### ISSUE THREAD

**Issue by wAsnk** [Lombiq/Orchard-Vue.js#145]
Building a clean repo will result in missing vue components in the Lombiq.VueJs.dll and also it will do the same in projects that are using this module.

Repo steps:
1. Delete wwwroot folder, or use a clean repository.
2. Build the project
3. The Lombiq.VueJs.dll is missing vue folder and the vue components.
4. Build the project again (The wwwroot already contains all the compiled scripts for vue component)
5. The Lombiq.VueJs.dll has all the components.

I used [ILSpy](https://github.com/icsharpcode/ILSpy) to check dll resources.

The vue components should be compiled to the wwwroot folder before the assembly and the Lobiq.VueJs.dll is created.

[Jira issue](https://lombiq.atlassian.net/browse/OSOE-892)

**Comment by Piedone** [comment]
You included a solution here: https://github.com/Lombiq/Orchard-Vue.js/pull/144 Isn't that suitable long-term?

**Comment by sarahelsaig** [comment]
Sounds like a build order problem?

**Comment by wAsnk** [comment]
> You included a solution here: #144 Isn't that suitable long-term?

No, that only moves the script files, but it does not solve the issue with vue components. The vue component are in the Assets/Scripts/VueComponents folder, and they require a special compilation process.

> Sounds like a build order problem?

Yes

**Comment by Piedone** [comment]
And/or the same thing that we solved in Node.js Extensions somehow (check targets and props files), since static files built be NE weren't included in the assembly either.

**Comment by Piedone** [comment]
Something similar was fixed here too: https://github.com/Lombiq/NPM-Targets/pull/41.

**Comment by Piedone** [comment]
Since the `Lombiq.VueJs` project is now deprecated, we won't do this. Note that we continue to use and support `Lombiq.VueJs.Resources`.
